src/kitovu/cli.py

```diff
@@ -44,8 +44,15 @@ def cli(loglevel: str) -> None:
 @cli.command()
 def gui() -> None:
     """Start the kitovu GUI."""
-    from kitovu.gui import app as guiapp
-    sys.exit(guiapp.run())
+    try:
+        from kitovu.gui import app as guiapp
+        sys.exit(guiapp.run())
+    except ModuleNotFoundError as e:
+        if e.name == 'PyQt5':
+            print('To run the GUI, you need to install the extra GUI dependencies')
+            print('Run "pip install kitovu[gui]"')
```

Comment:
Add `, file=sys.stderr` to both, so this goes to standard error instead of standard output.

Also, add `sys.exit(1)` so the exit status makes it clear that there was a failure.

Reply:
We should probably quote the `kitovu[gui]` to make things clearer - probably using double-quotes though, so it also works on Windows.

What about:

`'To do so, run: pip install "kitovu[gui]"'`